examples/carla/carla_simulator.py

```diff
@@ -21,32 +24,38 @@
 
 @dataclass
 class CarlaSimulationConfig:
-    npc_bps: Tuple[str] = NPC_BPS
-    roi_center: cord = cord(x=0, y=0)  # region of interest center
-    map_name: str = "Town03"
-    scene_name: str = "CARLA:Town03:Roundabout"
-    fps: int = 10  # Should not be compatible with invertedAI fps
-    traffic_count: int = 20
-    episode_length: int = 20  # In Seconds
-    entrance_interval: int = 2  # In Seconds
-    follow_ego: bool = False
-    slack: int = 3
-    ego_bp: str = "vehicle.tesla.model3"
-    seed: float = time.time()
-    flag_npcs: bool = True
-    flag_ego: bool = True
-    ego_autopilot: bool = True
-    npcs_autopilot: bool = False
-    populate_npcs: bool = True
-    npc_population_interval: int = 1  # In Seconds
+    """
+    A collection of static configuration options for the CARLA simulation.
+    Three different settings for handling NPCs outside the supported area available:
+    - "no_non_roi_npcs" - destroy NPCs leaving the supported area
+    - "spawn_at_entrace" - as above, but periodically spawn replacements within the supported area
+    - "calra_handoff" - hand off NPCs to CARLA's traffic manager when they leave supported area
+    """
+    ego_bp: str = "vehicle.tesla.model3"  #: blueprint name for the ego vehicle
+    npc_bps: Tuple[str] = NPC_BPS  #: blueprint names for NPC vehicles
+    location: str = "CARLA:Town03:Roundabout"  #: in format recognized by Inverted AI API
+    fps: int = 10  #: 10 is the only value currently compatible with Inverted AI API
+    traffic_count: int = 20  #: total number of vehicles to place in simulation
+    episode_length: int = 20  #: in seconds
+    follow_ego: bool = False  #: whether the spectator camera follows the ego vehicle
+    slack: int = 3  #: in meters, how far outside supported area to track NPCs
+    seed: float = time.time()  #: random number generator seed to control stochastic behavior
+    flag_npcs: bool = True  #: whether to display a blue dot on Inverted AI NPCs
+    flag_ego: bool = True  #: whether to display a red dot on the ego vehicle
+    ego_autopilot: bool = True  #: wheter to use traffic manager to control ego vehicle
+    npcs_autopilot: bool = False  #: ???
```

Comment:
I couldn't figure out what happens if I set it to `True`. Can you either explain that here or remove this option?

Reply:
If it's True, NPCs can be controlled by CARL. I had it initially to test the environment but kept it in case we need it in the future. I can either remove it or add a comment for it

Reply:
probably best to remove it then